.github/workflows/checks.yml

```diff
@@ -103,6 +103,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Checkout ITDE
+        run: git clone https://github.com/exasol/integration-test-docker-environment.git
+        working-directory: ..
+
+      - name: Start EXASOL Test-Environment
+        run: ./start-test-env spawn-test-environment --environment-name test --database-port-forward 8888 --bucketfs-port-forward 6666 --db-mem-size 4GB
+        working-directory: ../integration-test-docker-environment
```

Comment:
I think, we don't need to start the itde here, pytest_itde should be able to do this for us, if you are using the following fixture https://github.com/exasol/pytest-plugins/blob/dd03777bea7aed0dcaecb426469716d272da4dd7/pytest-itde/exasol/pytest_itde/__init__.py#L211C5-L211C9. However, we can also change this in another PR, if this is better for you.